tools/mason/MasonExternal.chpl

```diff
@@ -41,7 +43,24 @@ proc masonExternal(args: [] string) {
       exit(0);
     }
     else if args[2] == "--setup" {
-      if isDir(SPACK_ROOT) then
+      if !isDir(getSpackRegistry) && isDir(SPACK_ROOT) then {
```

Comment:
I'll only mention this once, but you can apply elsewhere: `then` is only needed when omitting the braces, e.g.

```chpl
if true then writeln('Ankush is a great student!');
```

```suggestion
      if !isDir(getSpackRegistry) && isDir(SPACK_ROOT)  {
```